plugin/src/helpers/utils.ts

```diff
@@ -174,3 +174,14 @@ export const findModuleFromBase = ({ paths, candidates }): string | null => {
   }
   return null
 }
+
+export const isNextAuthInstalled = (): boolean => {
+  try {
+    // eslint-disable-next-line import/no-unassigned-import, node/no-extraneous-require, import/no-extraneous-dependencies
+    require('next-auth')
+  } catch {
+    // Ignore the MODULE_NOT_FOUND error
+    return false
+  }
+  return true
```

Comment:
[dust] Consider returning `true` right after `require('next-auth')` instead of after the catch block.